File: translations/codenet/uncommented/C2J/deepseek/Code/s858795471.java
public class Main {
    public static boolean search(int[] A, int n, int key) {
        int i = 0;
        A[n] = key;
        while (A[i] != key) i++;
        return i != n;
    }

    public static void main(String[] args) {
        java.util.Scanner scanner = new java.util.Scanner(System.in);
        int n = scanner.nextInt();
        int[] A = new int[10000 + 1];
        for (int i = 0; i < n; i++) {
            A[i] = scanner.nextInt();
        }
        int q = scanner.nextInt();
        int sum = 0;
        for (int i = 0; i < q; i++) {
            int key = scanner.nextInt();
            if (search(A, n, key)) sum++;
        }
        System.out.println(sum);
    }
}

File: translations/codenet/uncommented/C2Go/codellama/Code/s858795471.go
package main

import "fmt"

func search(A []int, n int, key int) bool {
    A = append(A, key)
    for i := 0; i < n; i++ {
        if A[i] == key {
            return true
        }
    }
    return false
}

func main() {
    var n, q, key, sum int
    var A []int

    fmt.Scan(&n)
    A = make([]int, n)
    for i := 0; i < n; i++ {
        fmt.Scan(&A[i])
    }
    fmt.Scan(&q)
    for i := 0; i < q; i++ {
        fmt.Scan(&key)
        if search(A, n, key) {
            sum++
        }
    }
    fmt.Println(sum)
}

